packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -74,6 +74,14 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
     metadata: mergeMetadata({ ...input.metadata }),
   };
 
+  if (input.logo) {
+    data.logo = input.logo;
+    data.logoUrl = await uploadLogo({
+      logo: input.logo,
+      teamId: currentOrgId,
+    });
+  }
+
```

Comment:
There should be an option to remove the logo also like we do in profile page of user

Reply:
the UI already has remove button so you just have to add logic here

Reply:
Good spot, missed the else in case of a "null" value.